### DISCUSSION TITLE
osumania health drain

### DISCUSSION THREAD

**Discussion by JARDIDS** [ppy/osu#16585] [Ideas] [answered]
why does osumania have osustandard's health drain like now it makes osumania harder to play like take for example fnf matt's beatmap or other beatmaps that have insane health drain settings it'll make it impossible to play so can you remove it?

**Comment by peppy** [comment, marked as answer]
you can turn in "no fail" mod if you can't manage the health drain. for more information on the current situation try searching, there's plenty of ongoing discussion. we also did a community meeting centered around this discussion if you wish to listen.

**Comment by smoogipoo** [comment]
1. I don't know what you mean by "fnf matt's beatmap". There are multiple such beatmaps, and I'd need a specific one to see if there's anything to action upon.
2. Your usage of "it'll" sounds like you haven't tried it. I'd need to see a replay or a video showing you failing unreasonably.
3. Nothing is "impossible", the system is _designed_ for that to not be the case. If something's "impossible" that's only a skill issue. It may be "hard", to a certain degree, but you shouldn't notice much/any difference if you're playing the map at a good skill level (let's say >=90% acc).
